pandas/tests/tslibs/test_conversion.py

```diff
@@ -11,13 +11,17 @@
     timezones,
     tzconversion,
 )
+from pandas.compat import PY39
 
 from pandas import (
     Timestamp,
     date_range,
 )
 import pandas._testing as tm
 
+if PY39:
+    from zoneinfo import ZoneInfo
```

Comment:
there's also a backport i think

Reply:
@jbrockmendel , thanks!

Added the backport. Since this is my first time messing with the dev requirements, please tell me if I missed something:
* added `backports.zoneinfo` to environment.yml
* ran the hook to mirror the addition in the main pip requirements
* added it to all the CI pipeline yml files. For MacOS and numpy dev added it to the pip section, since it wasn't available on conda
* added it separately to the py37_32bit setup script

Reply:
once you get this to work, would have to add this as a required dep for < 3.9 (which is prob ok to do).

Reply:
@jreback, hello! It certainly wouldn't hurt to add it, but it shouldn't be necessary if I manage to do it right. The idea was to use this issue to move away from relying on `tz` internals to get stuff done and to use only the public interface, which is the same between `tzinfo` implementations.

To be frank, last time I tried to tackle this, I got bogged down in the amount of rewriting that needs to get done with how barren base `tzinfo` is, but hopefully I'll have better luck this time. If not, and if we determine that this approach isn't viable, then we will need to add `backports.tzinfo` as a dependency and rely on zoneinfo specifics as another `tzinfo` implementation that we support.